flake.nix

```
@@ -0,0 +1,49 @@
+{
+  description = "Haskell Tiny Game Jam";
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+
+  outputs = { self, nixpkgs, flake-utils }:
+  flake-utils.lib.eachDefaultSystem (system:
+  let
+    pkgs = import nixpkgs { inherit system; };
+    hPkgs = pkgs.haskell.packages.ghc926; # need to match Stackage LTS version
+
+    myDevTools = [
+      pkgs.zlib
+      pkgs.libGL
+      pkgs.libGLU
+      pkgs.freeglut
+      hPkgs.ghc # GHC compiler in the desired version (will be available on PATH)
```

Perhaps common libs could be provided with:

```suggestion
      (hPkgs.ghcWithPackages (p: [p.ansi-terminal-game p.gloss])) # GHC compiler in the desired version (will be available on PATH)
```

---

will try

---

updated